scipy/special/_logsumexp.py

```diff
@@ -110,8 +110,25 @@ def logsumexp(a, axis=None, b=None, keepdims=False, return_sign=False):
     axis = tuple(range(a.ndim)) if axis is None else axis
 
     if xp_size(a) != 0:
+        with np.errstate(over='ignore'):
```

Comment:
```suggestion
        with np.errstate(divide='ignore', invalid='ignore', over='ignore'):
```

I think this fix the test failures.